examples/index.md

```diff
@@ -124,6 +124,43 @@ POST https://api.shipcloud.io/v1/shipments
 }
 {% endhighlight %}
 
+### DHL - Visual age check
+When sending goods that can only legally be bought by people starting from a specific age you can
```

Comment:
I thinks this sound a bit more rounded:

"When sending goods that are only legally available for people of a specific age, you can request the carrier to check the receiver's age visually. Just add the additional service `visual_age_check` and either `16` or 18 as its minimum_age value."